pyquil/api/_qpu.py

```diff
@@ -114,7 +113,6 @@ def __init__(
         priority: int = 1,
         timeout: float = 10.0,
         client_configuration: Optional[QCSClientConfiguration] = None,
-        engagement_manager: Optional[EngagementManager] = None,
```

Comment:
Lets makes sure the commit captures this as a breaking change.

Reply:
Updated the MR title to `BREAKING CHANGE`